be/CMakeLists.txt

```diff
@@ -1064,7 +1061,7 @@ if (NOT ("${MAKE_TEST}" STREQUAL "ON" AND "${BUILD_FOR_SANITIZE}" STREQUAL "ON")
 endif()
 
 set(STARROCKS_LINK_LIBS ${STARROCKS_LINK_LIBS}
-    ${WL_LINK_STATIC} -lbfd
+    ${WL_LINK_STATIC} -lgomp -lbfd
```

Comment:
what does this library from? from gcc static link library?

Reply:
yes